docs/dev_mode.md

```diff
@@ -8,33 +8,19 @@ Arbitrary agent services built with the {{open_autonomy}} framework can be run w
 
 ### Environment setup
 
-Export the following environment variables (where you replace the paths with valid ones):
-```bash
-export OPEN_AEA_REPO_PATH="/path/to/open-aea-repo"
-export DEPLOYMENT_TYPE="docker-compose"
-export DEPLOYMENT_SPEC="/path/to/deployment-spec"
-export VERSION=dev
-```
-
-The default deployment specifications are stored in `deployments/deployment_specifications`.
-
-The `open-aea` repository can be cloned from here: https://github.com/valory-xyz/open-aea
+Fetch the required service or if you already have a service defined navigate to the service directoy and run
 
-*Tip: You can also store above variables in your .bashrc file*
-
-
-### Prepare images & run
-
-Execute the following:
 ```bash
-make build-deploy
+# build dev deployment using
+$ autonomy deploy build --dev --packages-dir PATH_TO_PACKAGES_DIR --open-aea-dir PATH_TO_LOCAL_OPEN_AEA_REPO --open-autonomy-dir PATH_TO_LOCAL_OPEN_AUTONOMY_DIR
```

Comment:
So we need 3 different directories? Does the user need to clone both open-aea and open-autonomy then?

Reply:
Yeah, if someone wants to use the dev mode they need to clone the open-aea and open-autonomy repo but not the packages dir. The packages dir is supposed to point at the local repo where they store their dev packages.